android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiNestedListView.java

```diff
@@ -0,0 +1,448 @@
+					return;
+				}
+
+				// Scroll the ListView by the y-axis amount given to us by the child view.
+				if (Build.VERSION.SDK_INT >= 19) {
+					ApiLevel19.scrollListBy(this, dyUnconsumed);
+				}
+			}
+
+			/**
+			 * Called by a NestedScrollingChild after a call to onStartNestedScroll() has returned true.
+			 * This indicates that the child will be dispatching scroll/fling events to this parent view.
+			 * @param child
+			 * Reference to either the NestedScrollingChild or the ViewGroup owned by this parent view
+			 * that contains the NestedScrollingChild.
+			 * @param target The nested scrollable child that is making this request.
+			 * @param axes
+			 * Bit mask set to flags ViewCompat.SCROLL_AXIS_HORIZONTAL and/or
+			 * ViewCompat.SCROLL_AXIS_VERTICAL that the child wants to scroll the parent by.
+			 */
+			@Override
+			public void onNestedScrollAccepted(View child, View target, int axes)
+			{
+				this.nestedParentHelper.onNestedScrollAccepted(child, target, axes);
+			}
+
+			/**
+			 * Called by a NestedScrollingChild when its startNestedScroll() method has been called.
+			 * This asks this view if it supports nested scrolling with along the given axes.
+			 * @param child
+			 * Reference to either the NestedScrollingChild or the ViewGroup owned by this parent view
+			 * that contains the NestedScrollingChild.
+			 * @param target The nested scrollable child that has been started.
+			 * @param axes
+			 * Bit mask set to flags ViewCompat.SCROLL_AXIS_HORIZONTAL and/or
+			 * ViewCompat.SCROLL_AXIS_VERTICAL that the child wants to scroll the parent by.
+			 * @return
+			 * Returns true if this view is will to scroll in the direction(s) indicated by argument "axes".
+			 * Returns false if unwilling to do the requested nested scrolling.
+			 */
+			@Override
+			public boolean onStartNestedScroll(View child, View target, int axes)
+			{
+				// Only return true if the NestedScrollingChild view wants to scroll this view vertically.
+				// Note: This is because the Android ListView class only supports vertical scrolling.
+				return ((axes & ViewCompat.SCROLL_AXIS_VERTICAL) != 0);
+			}
+
+			/**
+			 * Called by a NestedScrollingChild when its stopNestedScroll() method has been called.
+			 * @param target The nested scrollable child that has been stopped.
+			 */
+			@Override
+			public void onStopNestedScroll(View target)
+			{
+				this.nestedParentHelper.onStopNestedScroll(target);
+			}
+		}
+	}
+
+	/** Provides access to API Level 19 features. */
+	private static class ApiLevel19
+	{
+		/** Constructor made private to prevent instances from being made. */
+		private ApiLevel19() {}
+
+		/**
+		 * Calls the given ListView class' scrollListBy() method.
+		 * @param listView The list view to be scrolled. Can be null, in which case this method will no-op.
+		 * @param y The y-axis distance in pixels to scroll the list view by.
+		 */
+		public static void scrollListBy(ListView listView, int y)
```

Comment:
I'm not sure of the benefit of this since there are a lot of `dalvikvm` reference warnings from the support libraries themselves. But since it's here, why not?